### PR TITLE
Replace list with glob for vdirs

### PR DESCRIPTION
**Pull request by youngmoney**
This is #419, but I accidently closed 419, so here it is

### CODE REVIEW

**Comment by untitaker** [comment]
Do we really need recursive behavior then if we have globs?

**Comment by youngmoney** [comment]
I'm happy to remove it. I left it in simply because I don't want this update to ruin everyones config.

**Comment by untitaker** [comment]
Let's wait for a response from @geier though ^^

**Comment by geier** [comment]
I don't mind removing the recursion as long as its documented.

**Comment by youngmoney** [comment]
I've added a second commit without recursive, I can squash them or leave them, just let me know

File: CHANGELOG.rst [hunk 1]
```diff
@@ -14,7 +14,8 @@ not released yet
 
 * removed configuration variable `encoding` (in section [locale]), the correct
   locale should now be figured out automatically (Markus Unterwaditzer)
-* added command `list`
+* added command `list` (Taylor Money)
+* calendar path is now a glob without recursion (Taylor Money)
```
Comment (geier) [inline CHANGELOG.rst]: but only in the case of `type = discover`, right?

Comment (youngmoney) [inline CHANGELOG.rst]: correct, I've updated the wording in the changelog and the docs

**Comment by geier** [comment]
Looks good. Sorry that I took so long to review this that master has evolved again and we cannot cleanly apply this.
@younmoney, I have added you as a collaborator to khal, but please continue to send PRs for larger changes, especially any that break the UI.

Feel free to merge this (I don't mind if you rebase and then merge or directly merge).

Thanks for all your work!

**Comment by geier** [comment]
Looks good. Sorry that I took so long to review this that master has evolved again and we cannot cleanly apply this.
@younmoney, I have added you as a collaborator to khal, but please continue to send PRs for larger changes, especially any that break the UI.

Feel free to merge this (I don't mind if you rebase and then merge or directly merge).

Thanks for all your work!

**Pull request merged**